x/evm/keeper/params.go

```diff
@@ -21,3 +21,38 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// GetChainConfig returns the chain configuration parameter.
+func (k Keeper) GetChainConfig(ctx sdk.Context) types.ChainConfig {
+	chainCfg := types.ChainConfig{}
+	k.paramSpace.GetIfExists(ctx, types.ParamStoreKeyChainConfig, &chainCfg)
+	return chainCfg
+}
+
+// GetEVMDenom returns the EVM denom.
+func (k Keeper) GetEVMDenom(ctx sdk.Context) string {
+	evmDenom := ""
```

Comment:
this is just a nit pick but to maintain a similar structure to the rest of the repo this could be `var evmDenom string`